.github/workflows/main.yml

```diff
@@ -7,39 +7,31 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  test:
+  test-pip:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 30
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04 , windows-latest, macos-latest]
+        os: [ubuntu-latest, windows-latest, macos-latest]
         qt-lib: [pyqt, pyside]
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10"]
         include:
-          - python-version: "3.7"
-            qt-lib: "pyqt"
-            qt-version: "PyQt5~=5.12.0"
-            numpy-version: "~=1.18.0"
-          - python-version: "3.7"
-            qt-lib: "pyside"
-            qt-version: "PySide2~=5.12.0"
-            numpy-version: "~=1.18.0"
           - python-version: "3.8"
             qt-lib: "pyqt"
-            qt-version: "PyQt5~=5.15.0"
+            qt-version: "PyQt5~=5.12.0"
             numpy-version: "~=1.19.0"
           - python-version: "3.8"
             qt-lib: "pyside"
-            qt-version: "PySide2~=5.15.0"
+            qt-version: "PyQt5~=5.15.0"
```

Comment:
qt-lib: "pyside" should be using a PySide binding

Reply:
🤦🏻

Reply:
Fixed now, nice catch.